src/editor/initialize.ts

```diff
@@ -20,6 +21,10 @@ export class FolderInitializer {
     }
 
     async showDialog() {
+        if (shouldShowNotifications()) {
```

Comment:
Early return if notifications are enabled? Shouldn't the condition be negated?

Reply:
Whoops, fixed.